src/Home.js

```diff
@@ -1,17 +1,111 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, Text, View, FlatList, Pressable } from 'react-native';
+import { DataStore } from 'aws-amplify';
+import { Todo } from './models';
 
 export default function Home() {
+
+    const [todos, setTodos] = useState([]);
+
+    let subscription;
```

Comment:
Just curious here, can you explain to me why we cannot just define `subscription` as a `const` on ln 19 and remove the conditional reassignment?

Reply:
Thats a good point. I copied it from the JS library but your point is valid. I tested it out and it works being defined in the `useEffect`. Thats the only place we are using it.

Reply:
Regarding `msg`, I think it is a good idea to leave it there even though we are not using it. It will allow them to easily know, they can receive the updated Todo Object here and add some more functionality if needed.

I updated the `const onQuery`. That was a good catch.